docs/source/_static/custom.css

```diff
@@ -27,4 +35,380 @@
+    color: #e5e5e5;
+    box-shadow: 0 1px 4px rgba(230, 163, 211, 0.08);
+}
+
+/* Links: colored, but no underline */
+a {
+    text-decoration: none;
+    transition: color 0.2s;
+}
+
+/* Light theme links */
+[data-theme="light"] a {
+    color: #a72264;
+}
+
+[data-theme="light"] a:hover,
+[data-theme="light"] a:focus {
+    color: #7d174b;
+    background: #f7e6f6;
+}
+
+/* Dark theme links */
+[data-theme="dark"] a {
+    color: var(--color-brand-primary);
+}
+
+[data-theme="dark"] a:hover,
+[data-theme="dark"] a:focus {
+    color: #f0b3e0;
+    background: #3a2a3a;
+}
+
+/* Table styling with gradient background */
+table.docutils {
+    border-radius: 8px;
+    overflow: hidden;
+    border-collapse: separate;
+    border-spacing: 0;
+}
+
+/* Light theme tables */
+[data-theme="light"] table.docutils {
+    background: linear-gradient(120deg, #fff 80%, #f7e6f6 100%);
+    box-shadow: 0 1px 8px rgba(167, 34, 100, 0.06);
+}
+
+[data-theme="light"] table.docutils th,
+[data-theme="light"] table.docutils td {
+    border: 0.2px solid #ececec;
+    border-left: none;
+    border-right: none;
+}
+
+[data-theme="light"] table.docutils th {
+    background: #f0e6f6;
+    color: #a72264;
+    font-weight: 600;
+    border-top: 1px solid #ececec;
+    border-bottom: 1px solid #ececec;
+}
+
+/* Dark theme tables */
+[data-theme="dark"] table.docutils {
+    background: linear-gradient(120deg, #2a2a2a 80%, #3a2a3a 100%);
+    box-shadow: 0 1px 8px rgba(230, 163, 211, 0.12);
+}
+
+[data-theme="dark"] table.docutils th,
+[data-theme="dark"] table.docutils td {
+    border: 0.2px solid #404040;
+    border-left: none;
+    border-right: none;
+    color: #e5e5e5;
+}
+
+[data-theme="dark"] table.docutils th {
+    background: #3a2a3a;
+    color: var(--color-brand-primary);
+    font-weight: 600;
+    border-top: 1px solid #404040;
+    border-bottom: 1px solid #404040;
+}
+
+/* Improve spacing for lists */
+ul, ol {
+    margin-bottom: 1.2em;
+    padding-left: 2em;
+}
+
+/* Add subtle fade-in animation for main content */
+@keyframes fadeIn {
+    from { opacity: 0; transform: translateY(16px); }
+    to { opacity: 1; transform: none; }
+}
+
+main, .content, .document {
+    animation: fadeIn 0.7s cubic-bezier(.4,0,.2,1);
+}
+
+/* Responsive tweaks for mobile */
+@media (max-width: 700px) {
+    .sidebar-version {
+        font-size: 0.95rem;
+        padding: 0.7em 1em;
+    }
+    h1 {
+        font-size: 2rem;
+    }
+    h2 {
+        font-size: 1.3rem;
+    }
+    .highlight, pre, code {
+        font-size: 0.95em;
+    }
+}
+
+/* Custom scrollbar for a modern touch */
+::-webkit-scrollbar {
+    width: 8px;
+}
+
+/* Light theme scrollbar */
+[data-theme="light"] ::-webkit-scrollbar {
+    background: #f0e6f6;
+}
+
+[data-theme="light"] ::-webkit-scrollbar-thumb {
+    background: #a72264;
+    border-radius: 8px;
+}
+
+/* Dark theme scrollbar */
+[data-theme="dark"] ::-webkit-scrollbar {
+    background: #2a2a2a;
```

Comment:
Minor: There are a few colors that would be great if it is declared as variables.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_cascading_variables/Using_CSS_custom_properties